frontend/src/components/Auth/Login/Login.js

```diff
@@ -0,0 +1,75 @@
+import "../Auth.scss";
+import React, { useEffect, useState } from "react";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import { Link } from "react-router-dom";
+import useForm from "../../../Hooks/useForm";
+
+const onSubmit = event => {
+  event.preventDefault();
+};
+
+const Login = () => {
+  const [form, setForm] = useForm({
+    email: {
+      invalid: true,
+      touched: false,
```

Comment:
Since all the items you pass have the same properties, how about just passing list of key names like `['email', 'password']` and create the initial object in the hook. Will cut a lot of code